src/openrct2/interface/Screenshot.cpp

```diff
@@ -87,6 +87,20 @@ static sint32 screenshot_get_next_path(char *path, size_t size)
     rct2_time currentTime;
     platform_get_time_local(&currentTime);
 
+#ifdef _WIN32
+	// On NTFS filesystems, a colon (:) in a path
+	// indicates you want to write a file stream
+	// (hidden metadata). This will pass the
+	// file_exists and fopen checks, since it is
+	// technically valid. We don't want that, so
+	// replace colons with hyphens in the park name.
+	char * foundColon;
+	while (NULL != (foundColon = strstr(park_name, ":")))
```

Comment:
As `:` is just a single character, why not use `strchr`?

You could also optimise this a little by initialising `foundColon` to `park_name` and use that as argument to `strchr`, that way you would skip all the characters that you already processed and know are colon-free.